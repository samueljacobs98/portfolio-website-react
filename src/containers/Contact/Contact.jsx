import Links from "../../components/Links/Links";
import "./Contact.scss";
import earth from "../../assets/svgs/earth.svg";

const Contact = () => {
  return (
    <div className="contact" id="contact">
      <h2 className="contact__title">Contact Me</h2>
      <a className="contact__anchor" href="mailto:sammyjacobscontact@gmail.com">
        Get in touch!
      </a>
      <img className="contact__earth" src={earth} alt="" />
      <Links />
    </div>
  );
};

export default Contact;
