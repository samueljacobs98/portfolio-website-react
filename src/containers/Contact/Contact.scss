@use "../../assets/scss/variables" as *;

.contact {
  position: relative;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  min-height: 100vh;

  &__anchor,
  &__anchor:visited {
    text-decoration: none;
    color: $colour-background;
    background-color: $colour-orange;
    padding: $padding-button;
    border-radius: $padding-button;
    margin-top: 20px;
    box-shadow: 5px 5px 5px rgb(0 0 0 / 50%);
  }
}
