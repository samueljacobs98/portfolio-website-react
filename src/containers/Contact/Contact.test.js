import { render } from "@testing-library/react";
import Contact from "./Contact";

test("should match snapshot", () => {
  const { container } = render(<Contact />);
  expect(container).toMatchSnapshot();
});
