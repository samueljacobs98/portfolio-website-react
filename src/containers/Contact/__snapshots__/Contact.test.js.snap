// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <div
    class="contact"
    id="contact"
  >
    <h2
      class="contact__title"
    >
      Contact Me
    </h2>
    <a
      class="contact__anchor"
      href="mailto:sammyjacobscontact@gmail.com"
    >
      Get in touch!
    </a>
    <img
      alt=""
      class="contact__earth"
      src="earth.svg"
    />
    <div
      class="links"
    >
      <a
        href="https://www.linkedin.com/in/samuel-j-66a84ba2/"
      >
        <img
          alt="link to linked in"
          class="links__icon"
          src="linkedin-icon.svg"
        />
      </a>
      <a
        href="https://github.com/samueljacobs98"
      >
        <img
          alt="link to github"
          class="links__icon"
          src="github-icon.svg"
        />
      </a>
      <a
        href="https://www.instagram.com/_samuel.jacobs_"
      >
        <img
          alt="link to instagram"
          class="links__icon"
          src="instagram-icon.svg"
        />
      </a>
      <img
        alt="lines leading to social links"
        class="links__lines"
        src="link-lines.svg"
      />
    </div>
  </div>
</div>
`;
